app/src/main/res/values/strings.xml

```diff
@@ -120,9 +120,10 @@
     <string name="marker_name">Marker name</string>
     <string name="save_poi">save</string>
     <string name="cancel_poi">Cancel</string>
+    <string name="some_permissions_missing">Location is not allowed. Change in settings</string>
 
     <!--Copyright-->
-    <string name="version_0_5">Version: 0.5</string>
+    <string name="version_0_9">Version: 0.9</string>
```

Comment:
Is there a special reason for updating it? I have no clue about copyright stuff (I don't think it's bad, I just don't know)

Reply:
I was following plan to update version from assignment to assignment